src/inference/include/openvino/runtime/properties.hpp

```diff
@@ -519,6 +519,8 @@ struct Priorities : public Property<std::string> {
  * @brief Device Priorities config option, with comma-separated devices listed in the desired priority
  */
 static constexpr Priorities priorities{"MULTI_DEVICE_PRIORITIES"};
+static constexpr Priorities batch_priorities{"AUTO_BATCH_DEVICE_CONFIG"};
```

Comment:
maybe this property name does not make sense(it's not priority related key), and it should fall in ov::hint category

Reply:
sorry I misunderstand the key meaning. Updated.